apps/src/templates/EligibilityChecklist.jsx

```diff
@@ -0,0 +1,198 @@
+    submission: {
+      name: '',
+      email: '',
+      role: '',
+      country: 'United States',
+      hoc: '',
+      nces: '',
+      schoolName: '',
+      schoolCity: '',
+      schoolState: '',
+      schoolZip: '',
+      schoolType: '',
+      afterSchool: '',
+      tenHours: '',
+      twentyHours: '',
+      otherCS: false,
+      followUpFrequency: '',
+      followUpMore: '',
+      acceptedPledge: false
+    },
+    errors: {
+      invalidEmail: false
+    }
+  };
+
+  // Saves the teaching-year choice to trigger next step of actions
+  handleSubmit = () => {
+    this.setState({ statusYear: (this.state.yearChoice ? SUCCEEDED : FAILED )});
+  }
+
+  // Temporarily saves the teaching-year
+  // Param: accept - boolean - true for years to accept, false for deniable answers
+  handleYearChange = (accept) => {
+    this.setState({yearChoice : accept});
+  }
+
+  handleDropdownChange = (field, event) => {
+    this.setState({
+      submission: {
+        ...this.state.submission,
+        [field]: event ? event.value : ''
+      }
+    });
+  }
+
+  handleNotFoundChange = (field, event) => {
+    this.setState({
+      submission: {
+        ...this.state.submission,
+        [field]: event.target.value
+      }
+    }, this.checkShowFollowUp);
+  }
+
+  render() {
+    const {submission, errors} = this.state;
+    return (
+      <div>
+        <h2>
+          {i18n.eligibilityRequirements()}
+        </h2>
+        <div>
+          {i18n.eligibilityExplanation()}
+        </div>
+        <SetupStep
+          stepName={i18n.eligibilityReqPD()}
+          stepStatus={this.props.statusPD}
+        >
+          {i18n.eligibilityReqPDFail()}
+        </SetupStep>
+        <SetupStep
+          stepName={i18n.eligibilityReqStudentCount()}
+          stepStatus={this.props.statusStudentCount}
+        >
+          {i18n.eligibilityReqStudentCountFail()}
+        </SetupStep>
+        {/* Short version - displayed while 'focus' on other eligibility requirements */}
+        {this.props.statusStudentCount !== SUCCEEDED &&
+          <SetupStep
+            stepName={i18n.eligibilityReqYear()}
+            stepStatus={this.state.statusYear}
+          />
+        }
+        {/* Long version - displayed while 'focus' on this eligibility requirements */}
+        {this.props.statusStudentCount === SUCCEEDED &&
+          <div>
+            <SetupStep
+              stepName={i18n.eligibilityReqYear()}
+              stepStatus={this.state.statusYear}
+              displayExplanation={true}
+            >
+              {i18n.eligibilityReqYearFail()}
+            </SetupStep>
+            <div>
+              <b>{i18n.eligibilityReqYearConfirmInstructions()}</b>
+              <div>
+                <form>
+                  <label>
+                    <input type="radio" name="year" value="no" onChange={() => {this.handleYearChange(false);}} disabled={this.state.statusYear !== UNKNOWN ? true : false}/>
```

Comment:
```
disabled={this.state.statusYear !== UNKNOWN ? true : false}
```
can be
```
disabled={this.state.statusYear !== UNKNOWN}
```
This will already be true/false